### ISSUE TITLE
Help commands should be a bit smarter

### ISSUE THREAD

**Issue by naschorr** [naschorr/clipster#3]
If you mess up trying to invoke a help command, let's say '=\<category\>' instead of '=help \<category\>', as long as '\<category\>' doesn't map to an existing command, it should be smart enough to dump the help interface for that command.

Clips.py already tracks category names inside Clips > self.clip_groups. So it should be fairly easy to implement.

**Comment by naschorr** [comment]
Closing, no longer relevant after #64
